packages/core/src/routes/interaction/index.test.ts

```diff
@@ -87,7 +87,7 @@ describe('session -> interactionRoutes', () => {
         })
       ),
     });
-    const path = `${interactionPrefix}/${verificationPrefix}/passcode`;
+    const path = `${interactionPrefix}${verificationPrefix}/passcode`;
```

Comment:
nit: So where did the backslash go? Is it contained in the `verificationPrefix` variable? Can we get it back as it reads slightly better. lol

Reply:
hmm good point. verification is noloner a prefix, let me update.

Reply:
updated